src/main/java/ch/njol/skript/sections/SecConditional.java

```diff
@@ -118,21 +118,35 @@ public boolean init(Expression<?>[] exprs,
 		multiline = parseResult.regexes.size() == 0 && type != ConditionalType.ELSE;
 
 		// ensure this conditional is chained correctly (e.g. an else must have an if)
-		SecConditional lastIf;
+		SecConditional precedingIf;
```

Comment:
There is not a reason for this variable to be declared here right? It is only used once later on but is recalculated before being used.